app/javascript/packs/application.js

```diff
@@ -13,8 +13,10 @@ window.Rails = Rails
 
 import 'bootstrap'
 import 'data-confirm-modal'
+import bsCustomFileInput from 'bs-custom-file-input'
 
 $(document).on("turbolinks:load", () => {
   $('[data-toggle="tooltip"]').tooltip()
   $('[data-toggle="popover"]').popover()
+  bsCustomFileInput.init()
```

Comment:
Do you think we need this library? I'd prefer to maybe not include extra dependencies and I don't think we need a fancy file input field for the base Jumpstart template. Maybe we could add this to the Wiki instead?

Reply:
Sounds good, I can make this change this week 👍